api/session_stop.go

```diff
@@ -0,0 +1,52 @@
+package api
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/canonical/lxd/lxd/response"
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/microcluster/v2/rest"
+	"github.com/canonical/microcluster/v2/state"
+
+	"github.com/canonical/microcloud/microcloud/api/types"
+	"github.com/canonical/microcloud/microcloud/service"
+)
+
+// SessionStopCmd represents the /1.0/session/stop API on MicroCloud.
+var SessionStopCmd = func(sh *service.Handler) rest.Endpoint {
+	return rest.Endpoint{
+		AllowedBeforeInit: true,
+		Name:              "session/stop",
+		Path:              "session/stop",
+
+		Put: rest.EndpointAction{Handler: authHandlerMTLS(sh, sessionStopPut(sh))},
+	}
+}
+
+// sessionStopPut receives a stop request from an initiator member and stops the current session.
```

Comment:
Theoretically the stop endpoint could also be invoked on the initiator so I would not be too specific in the comment and try to be generic by saying that is just stops the current session.